app/src/main/java/com/mobilecoronatracker/utils/DateUtils.kt

```diff
@@ -6,6 +6,12 @@ import java.util.TimeZone
 
 fun getTodayTimestamp(): Long = (System.currentTimeMillis() / MILLIS_PER_DAY) * MILLIS_PER_DAY
 fun getYesterdayTimestamp(): Long = getTodayTimestamp() - MILLIS_PER_DAY
+fun getTimestampForDaysBefore(daysCount: Int, referenceTimestamp: Long = getTodayTimestamp()): Long {
```

Comment:
Why can't we use a simpler approach like for yesterday?

Reply:
Definitely yes.